sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -367,6 +367,8 @@ export type RequestBodyType = NodeJS.ReadableStream | (() => NodeJS.ReadableStre
 // @public
 export class RestError extends Error {
     constructor(message: string, options?: RestErrorOptions);
+    // (undocumented)
+    [x: symbol]: () => string;
```

Comment:
huh how did this get here?

Reply:
I had a similar question, and I have no idea other than to keep CI happy.

Reply:
@jeremymeng what is the cause here?